arch/risc-v/src/esp32c3/esp32c3_rt_timer.c

```diff
@@ -212,15 +213,13 @@ static void stop_rt_timer(FAR struct rt_timer_s *timer)
       list_delete(&timer->list);
       timer->state = RT_TIMER_IDLE;
 
-      /* If timer is in in head of list */
+      /* If the timer is at the head of the list */
 
       if (ishead)
         {
-          /* If list is not empty */
-
           if (!list_is_empty(&s_runlist))
             {
-              /* Reset hardware timer alarm value to be next timer's */
+              /* Reset the hardware timer alarm value to be next timer's */
```

Comment:
I believe the current writing is correct, but here is a suggestion for a (maybe) more direct explanation:
```suggestion
              /* Set the value from next timer as the new hardware timer alarm value */
```